js/app.js

```diff
@@ -0,0 +1,91 @@
+import { printOutput, printDatesDiff } from "./output.js";
+import getDatesDiff from "./getDatesDiff.js";
+import { showContent } from "./showContent.js";
+import howl from "./howler.js";
```

Comment:
тебе же здесь переменная howl не нужна по сути, лучше вот так подключить howler
```suggestion
import "./howler.js";
```